XRTK-Core/Assets/XRTK.Tests/Core/TestFixture_02_MixedRealityToolkitUtilityTests.cs

```diff
@@ -16,56 +18,52 @@ namespace XRTK.Tests.Core
 {
     public class TestFixture_02_MixedRealityToolkitUtilityTests
     {
+        private void SetupServiceLocator()
```

Comment:
I think there is an attribute "TestPrepare" or something to setup a test before it runs.

Reply:
I think we should do a separate PR to review the Unit tests and add these features.  We need more unit tests too to be sure.

Reply:
Makes sense